### PR TITLE
docs: update stale references to audits

### PR DESCRIPTION
**Pull request by connorjclark**
missed these in #4982

### CODE REVIEW

File: docs/readme.md [hunk 1]
```diff
@@ -94,7 +94,7 @@ In order to extend the Lighthouse configuration programmatically, you need to pa
       'first-meaningful-paint',
       'speed-index-metric',
       'estimated-input-latency',
-      'first-interactive',
+      'first-cpu-idle',
       'consistently-interactive',
```
Comment (patrickhulce) [inline docs/readme.md]: should probably update `consistently-interactive`, `speed-index-metric`, and kill `estimated-input-latency` if we're updating these now, or could just wait for when we really kill EIL :)

**Comment by connorjclark** [comment]
> nice catch! not sure what spurs it now though :)

the golden lantern file has refernces to `timeToFirstInteractive`, which confused me.

File: docs/configuration.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ module.exports = {
       'first-meaningful-paint',
       'speed-index-metric',
```
Comment (patrickhulce) [inline docs/configuration.md]: oh sorry the same applied up here too!

Comment (patrickhulce) [reply docs/configuration.md]:
```suggestion
      'speed-index',
```

File: docs/configuration.md [hunk 3]
```diff
@@ -17,7 +17,7 @@ module.exports = {
...
       'estimated-input-latency',
-      'first-interactive',
+      'first-cpu-idle',
       'consistently-interactive',
```
Comment (patrickhulce) [inline docs/configuration.md]:
```suggestion
      'interactive',
```

File: docs/configuration.md [hunk 4]
```diff
@@ -151,7 +151,7 @@ The categories property controls how to score and organize the audit results in
       description: 'This category judges your performance',
       auditRefs: [
         {id: 'first-meaningful-paint', weight: 2, group: 'metrics'},
-        {id: 'first-interactive', weight: 3, group: 'metrics'},
+        {id: 'first-cpu-idle', weight: 3, group: 'metrics'},
         {id: 'consistently-interactive', weight: 5, group: 'metrics'},
```
Comment (patrickhulce) [inline docs/configuration.md]:
```suggestion
        {id: 'interactive', weight: 5, group: 'metrics'},
```

**Comment by brendankenny** [comment]
if we're updating should we just update to be consistent with the planned 6.0 set so we don't have to update again?

They're just examples, doesn't seem like we're losing anything if we don't keep EIL around until it's removed

**Pull request merged**